src/quickfix.c

```diff
@@ -4001,7 +4001,9 @@ qf_free_items(qf_list_T *qfl)
 		// TODO: Avoid qf_count being incorrect.
 		qfl->qf_count = 1;
 	}
-	qfl->qf_start = qfpnext;
+	// avoid using feed memory
+	if (!stop)
```

Comment:
As the above `if` statement already uses the `!stop` condition, can this be moved into the above if statement?

Reply:
sure, changed it